text/0000-label-rfcs.md

```diff
@@ -0,0 +1,35 @@
+# Meta
+[meta]: #meta
+- Name: Label RFCs With Specification And Target Audience
+- Start Date: 2020-08-15
+- Author(s): ForestEckhardt
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (N/A)
+
+# Summary
+[summary]: #summary
+
+It would be nice to start labeling RFCs with a label that indicates which part
+of the specification (i.e. Buildpack Interface, Distribution, Platform
+Interface) is being affected as well as a label to indicate the target audience
+(i.e. Buildpack Authors, CNB Contributors, Platform Operators) of the change.
+
+# Motivation
+[motivation]: #motivation
+
+Indicating which part of the specification the RFC is purposing to update or
+modify will make the RFC more discoverable because it will allow people to
+focus in on certain RFCs that effect them. This means that potentially effected
```

Comment:
```suggestion
focus in on certain RFCs that affect them. This means that potentially affected
```
https://www.grammarly.com/blog/affect-vs-effect/#:~:text=Here's%20the%20short%20version%20of,the%20result%20of%20a%20change.